diff --git a/cr3gui/src/cr3xcb.cpp b/cr3gui/src/cr3xcb.cpp
index 50c3702..1322e3c 100644
--- a/cr3gui/src/cr3xcb.cpp
+++ b/cr3gui/src/cr3xcb.cpp
@@ -76,7 +76,7 @@ typedef enum {
     DISCHARGING,
     NOT_CHARGING,
     FULL_CHARGE,
-    LOW_CHARGE,
+    LOW_CHARGE
 } battery_status_t;
 
 typedef struct {
diff --git a/cr3gui/src/settings.cpp b/cr3gui/src/settings.cpp
index fe240f9..0f3acf0 100644
--- a/cr3gui/src/settings.cpp
+++ b/cr3gui/src/settings.cpp
@@ -339,8 +339,9 @@ CRSettingsMenu::CRSettingsMenu( CRGUIWindowManager * wm, CRPropRef newProps, int
 		{NULL, NULL},
 	};
         item_def_t highlight_bookmark[] = {
-                {_("On"), "1"},
-                {_("Off"), "0"},
+                {_("None"), "0"},
+                {_("Solid"), "1"},
+                {_("Underline"), "2"},
                 {NULL, NULL},
         };
 
