diff -urBbN libunibreak-4.3-orig/CMakeLists.txt libunibreak-4.3/CMakeLists.txt
--- libunibreak-4.3-orig/CMakeLists.txt	1970-01-01 04:00:00.000000000 +0400
+++ libunibreak-4.3/CMakeLists.txt	2021-01-24 18:54:37.740977000 +0400
@@ -0,0 +1,32 @@
+# Custom lightweight cmake project to compile libunibreak as static library
+
+project(libunibreak C)
+
+cmake_minimum_required(VERSION 3.4)
+
+include (CheckTypeSize)
+include (CheckIncludeFile)
+include (CheckFunctionExists)
+
+
+CHECK_INCLUDE_FILE(stdlib.h HAVE_STDLIB_H)
+CHECK_INCLUDE_FILE(string.h HAVE_STRING_H)
+
+check_function_exists(memset HAVE_MEMSET)
+
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+include_directories(src)
+
+set(LIBUNIBREAK_SOURCE
+	src/unibreakbase.c
+	src/unibreakdef.c
+	src/linebreak.c
+	src/linebreakdata.c
+	src/linebreakdef.c
+	src/emojidef.c
+	src/graphemebreak.c
+	src/wordbreak.c
+)
+
+add_library(unibreak STATIC ${LIBUNIBREAK_SOURCE})
